config/config.ini

```diff
@@ -2,3 +2,35 @@
 governance_id = -1
 budget_id = -1
 
+[dev]
+snapshot_space = testnet-1.eth
+discord_vote_endtime = 120
+discord_yes_count_threshold = 1
+snapshot_url_prefix = https://testnet.snapshot.org/#/
+snapshot_hub = https://testnet.hub.snapshot.org
+network_id = 11155111
+settings_name = Testnet
+settings_about = Testnet settings for Snapshot space modification
+settings_symbol = GLD
+settings_members = 0x5B3eADC33DF96e51ba69907112f32eA863870FD4
+settings_strategies = [{"name": "erc20-balance-of", "network": "11155111", "params": {"symbol": "GLD", "address": "0xBdFc9BCEb963ef920DFA0237f72D83D2bBA5f17c", "decimals": 18}}]
+settings_token_addresses = 0xBdFc9BCEb963ef920DFA0237f72D83D2bBA5f17c
+primary_rpc_url = https://sepolia.drpc.org
+secondary_rpc_url = https://1rpc.io/sepolia
+
+[prod]
+snapshot_space = gov.bloomstudio.eth
+discord_vote_endtime = 172800
+discord_yes_count_threshold = 5
+snapshot_url_prefix = https://snapshot.org/#/
+snapshot_hub = https://hub.snapshot.org
+network_id = 42161
+settings_name = Bloom Studio
+settings_about = Bloom is a decentralized studio creating immersive, expressive games & other audiovisual experiences within a scalable, collective micro-economy
+settings_symbol = govXP
+settings_members = 0xd80e4ef6169e5f737430be8a845237befb9a0adb
+settings_strategies = [{"name": "erc20-balance-of", "network": "42161", "params": {"symbol": "uXP", "address": "0x57d3a929fdc4faf1b35e7092d9dee7af097afb6a", "decimals": 18}}, {"name": "erc20-balance-of", "network": "42161", "params": {"symbol": "aXP", "address": "0x206d247F61cb82B9711318381cDb7Bc5039d2A2c", "decimals": 18}}, {"name": "erc20-balance-of", "network": "42161", "params": {"symbol": "bXP", "address": "0x4cd06ada7d8564830018000d784c69bd542b1e6a", "decimals": 18}}]
+settings_token_addresses = 0x206d247F61cb82B9711318381cDb7Bc5039d2A2c,0x4cd06ada7d8564830018000d784c69bd542b1e6a,0x57d3a929fdc4faf1b35e7092d9dee7af097afb6a
+primary_rpc_url = https://arbitrum.llamarpc.com
```

Comment:
personally think these do better as secrets because it would allow us to use our own RPC with our private API keys

Reply:
> personally think these do better as secrets because it would allow us to use our own RPC with our private API keys

Are you only/specifically referring to the primary / secondary RPC vars? I think settings_strategies for example is OK in the config as it's a bit lengthy to be setting as an ENV var imo

Reply:
ya only RPC urls because instead of using public ones, we may want to use personal ones that include API keys